styles/login.module.scss

```diff
@@ -0,0 +1,37 @@
+.loginPage{
+  max-width: 400px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.loginPage label{
+  text-align: left;
+  display: block;
+
+}
+.loginPage h1{
+  font-size:  25px;
+  margin-bottom: 30px;
+}
+.loginPage h2{
+  font-size:  30px;
+}
+.loginPage input{
+  //text-align: center;
+  width:  100%;
+  padding:  6px 10px;
+  margin: auto;
+  border: 1px solid;
+  box-sizing: border-box;
+  display: block;
+  border-radius:  8px;
+}
+.loginPage button{
+  width:  100%;
+  background-color: coral;
+  color: white;
+  border: 0;
+  padding: 8px;
+  border-radius:  8px;
+  cursor: pointer;
+}
```

Comment:
add a new line here

Reply:
deleted the file in this branch. 
login.tsx is being reviewed in #39